tests/unit/test_hyper_transformer.py

```diff
@@ -128,53 +703,89 @@ def test_fit_transform(self):
             expect_call_args_transform
         )
 
-    def test__get_columns_one(self):
-        data = pd.DataFrame({
-            'a': [1, 2, 3],
-            'b': [1, 2, 3],
-        })
+    def test_reverse_transform(self):
+        """Test the ``reverse_transform`` method.
 
-        returned = HyperTransformer._get_columns(data, 'a')
+        Tests that ``reverse_transform`` loops through the ``_transformers_sequence``
+        in reverse order and calls ``transformer.reverse_transform``.
 
-        np.testing.assert_equal(returned, np.array(['a']))
+        Setup:
+            - The ``_transformers_sequence`` will be hardcoded with a list
+            of transformer mocks.
+            - The ``_output_columns`` will be hardcoded.
+            - The ``_temp_columns`` will be hardcoded.
 
-    def test__get_columns_two(self):
-        data = pd.DataFrame({
-            'a': [4, 5, 6],
-            'a#1': [7, 8, 9],
-            'b': [4, 5, 6],
-            'b#1': [7, 8, 9],
-        })
-
-        returned = HyperTransformer._get_columns(data, 'b')
-
-        expected = np.array([
-            'b',
-            'b#1',
-        ])
-        np.testing.assert_equal(returned, expected)
+        Input:
+            - A DataFrame of multiple types.
 
-    def test__get_columns_none(self):
+        Output:
+            - The reverse transformed DataFrame with the correct columns dropped.
+        """
+        # Setup
+        int_transformer = Mock()
+        int_out_transformer = Mock()
+        float_transformer = Mock()
+        categorical_transformer = Mock()
+        bool_transformer = Mock()
+        datetime_transformer = Mock()
         data = pd.DataFrame({
-            'a': [1, 2, 3],
+            'integer.out.value': [1, 2, 1, 3],
```

Comment:
Would it be worth it to pull out some of this input data / transformed data into a helper method? Seems like it's being reused to some extent.